tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_v1beta.yaml

```diff
@@ -0,0 +1,35 @@
+metadata:
+  description: |-
+    This test is to ensure that the workflow running in community mode
+    completes successfully on the RRFS_CONUScompact_25km grid using the RRFS_v1beta
+    physics suite with ICs and LBCs derived from the RRFS forecasts mapped onto
+    3-km regular grid (rrfs*.conus.grib2). This test uses old v1 sfc_data,
+    not the v2 fractional grid sfc_data.
```

Comment:
This comment does not look consistent with the config below. Please update it to accurately reflect the configured experiment.

Reply:
@christinaholtNOAA this PR does not yet use full RRFS files. It uses RRFS interpolated into 3-km conus grid. In order for the full RRFS files to be functional, new UFS_UTILS from Larissa's branch needs to be made and merged